lfs/client.go

```diff
@@ -125,8 +126,15 @@ func Download(oid string) (io.ReadCloser, int64, error) {
 		return nil, 0, err
 	}
 	LogTransfer("lfs.data.download", res)
+	
+	buf, _ := ioutil.ReadAll(res.Body)
+	body := myReader{bytes.NewBuffer(buf)}	
```

Comment:
This reads the body into memory, which could get pretty large. The body should be cleaned up by all callers of `Download()` instead.